### PR TITLE
doc: percentiles and hdr merge

### PR DESCRIPTION
**Pull request by sitano**
@busbey you asked for it (https://github.com/brianfrankcooper/YCSB/pull/1507#pullrequestreview-591345620), you get it ;)

This PR updates a README.md file with the information about how to work with latency percentiles and that a user shall not look up for averages.

### CODE REVIEW

**Comment by sitano** [comment]
@busbey ping

File: README.md [hunk 1]
```diff
@@ -78,3 +78,79 @@ To build the full distribution, with all database bindings:
 To build a single database binding:
 
     mvn -pl site.ycsb:mongodb-binding -am clean package
+
+Running multiple instances and latency percentiles
+--------------------------------------------------
+
+In general, you shall be interested in 99% percentile (P99) of the latency
+distribution and the rest - 99.9%, 99.99%, 99.999% - the latency tail.
+95% percentile has a very low probability to be observed by the user
```
Comment (allanbank) [inline README.md:87]: >  95% percentile has a very low probability to be observed by the user...

This was a little confusing to me at first.

Maybe reword to something like:

> A user is very unlikely to have _all_ of their requests happen within the 95% percentile, so don't even bother.

File: README.md [hunk 2]
```diff
@@ -78,3 +78,79 @@ To build the full distribution, with all database bindings:
...
+so don't even bother (see "How Many Nines?" at
+https://bravenewgeek.com/everything-you-know-about-latency-is-wrong/
+). The formula to calculate probability of how many clients will
+observe a specific percentile is: 
+
+    Probability_to_observe = 1 - Percentile ^ Requests
```
Comment (allanbank) [inline README.md:93]: Would this be clearer as:

> Probability_to_observe = 1 - Percentile<sup>Requests</sup>

File: README.md [hunk 3]
```diff
@@ -78,3 +78,79 @@ To build the full distribution, with all database bindings:
...
+    -p hdrhistogram.fileoutput=true
+    -p hdrhistogram.output.path=file.hdr
+
+merge them manually and extract required percentiles out of the
+joined result.
+
+Remember that running multiple workloads may distort original
+workloads distributions they were intended to produce.
+
+Merging HDR histogram percentiles
```
Comment (allanbank) [inline README.md:117]: I like that you pointed the user to tool to help them but I think it would be better to link them to the documentation for that tool rather than providing a synopsis here. As the HDRLogProcessing tool changes the likelihood of this documentation being updated is fairly low which will lead to questions and confusion for users.

Comment (sitano) [reply README.md:118]: I removed all examples and left the link only to the tool github which has a README. I left an example for CSV extraction. If you want ping me to remove it.

File: README.md (hunk 2 above)
Comment (sitano) [reply README.md:94]: I don't know how to do that and I did find a way to do that.

File: README.md (hunk 1 above)
Comment (sitano) [reply README.md]: I changed the wording to

> In general, you shall be interested in 99% percentile (P99) of the latency
distribution, and the rest of the tail - 99.9%, 99.99%, 99.999%. The difference
between the amount of requests that will be observed by a user that fall
into 95% (P95) percentile and 99% percentile may be sufficiently large.

> For example, see "How Many Nines?" at https://bravenewgeek.com/everything-you-know-about-latency-is-wrong/.
The formula to calculate probability of how many clients will observe
a specific percentile is:

**Comment by sitano** [comment]
@allanbank ping

**Review by busbey — APPROVED** [review]
thanks for the contribution, someone can redo the formulas with latex formatting later
